packages/perseus/src/widgets/radio/focus-ring.jsx

```diff
@@ -8,33 +8,28 @@ import * as React from "react";
 
 import * as styleConstants from "../../styles/constants.js";
 
-class FocusRing extends React.Component<{
-    children?: any,
+type FocusRingProps = {|
+    children: React.Node,
     // Whether the focus ring is visible. Allows for positioning
     // the child identically regardless of whether the ring is visible.
     visible: boolean,
     // Color of the focus ring
     color: string,
-    ...
-}> {
-    static defaultProps: {|color: string, visible: boolean|} = {
-        visible: true,
-        color: styleConstants.kaGreen,
```

Comment:
**suggsetion:** Dropping the default props affects the prop types that consumers of the component will see.  We should probably maintain these default props.  This can be done by setting
```
FocusRing.defaultProps = {
   visible: true,
   color: styleConstants.kaGreen,
};
```
You'll need to export the component after you define it though otherwise Flow complains about `defaultProps` not existing on `Function`, see the following [example](https://flow.org/try/#0JYWwDg9gTgLgBAKjgQwM5wEoFNkGN4BmUEIcARFDvmQNwBQdMAnmFnAArFjoC8cA3gB86cUQCNkUAFxxUMKMAB2AcwA0dQQF96dXBEVy4AMQgQ4fABRguqGZwjcAlDOx4YAOgByEACZseAHwCIqJwlDAArlCKcIoRADbx9Jp0JhDufgTICTD23OZwFvwS0uQAFliJEGSaMkIhcCUycgoq6lqOOnoGhKYAjAUAPGlwAPQB9N2GBKYATEMjJTxkAO7Q8T5kYxMMWAAekLACadp0QA).

Reply:
That's good to know, I didn't realize. Thanks!